services/QuillLMS/client/app/bundles/Connect/components/studentLessons/fillInBlank.tsx

```diff
@@ -80,7 +81,7 @@ export class PlayFillInTheBlankQuestion extends React.Component<PlayFillInTheBla
 
   setQuestionValues = (question: FillInBlankQuestion) => {
     const q = question;
-    const splitPrompt = q.prompt.split('___');
+    const splitPrompt = question.prompt.replace(/<\/p><p>/g, '</br>').replace(/^<p>/g, '').replace(/<p>/g, '<br/>').replace(/<\/p>/g, '').split('___');
```

Comment:
This function is implemented three times. Let's refactor it into a shared function. As a bonus, we'll also get to name it, which will add a bit of self documentation.